frontend/resources/styles/main/partials/dashboard.scss

```diff
@@ -236,7 +237,11 @@
       line-height: 1rem;
       font-weight: 600;
       color: $color-black;
-      margin-right: $size-4;
+      margin-right: $size-1;
+    }
+
+    .import-file {
```

Comment:
I don't understadn why this display:none is added here

Reply:
As import-file is a class given to form, which is never shown to the user as far as I know, so I thought it would be much better to make that import-file form hidden. and hence `display: none`.
Should I change it back to make it visible ??

Reply:
Yes please! :)